language/move-vm/natives/src/signature.rs

```diff
@@ -33,14 +32,14 @@ pub fn native_ed25519_publickey_validation(
     // This deserialization performs point-on-curve and small subgroup checks
     let valid = ed25519::Ed25519PublicKey::try_from(&key_bytes[..]).is_ok();
     let return_values = vec![Value::bool(valid)];
-    Ok(NativeResult::ok(cost, return_values))
+    NativeResult::ok(cost, return_values)
 }
 
 pub fn native_ed25519_signature_verification(
```

Comment:
Ooh, this is a nasty one. I think we can/should make this one abort-free; I will put up a separate PR for that.

Reply:
I'm not sure if I follow. This function cannot abort

Reply:
Was reading a comment for the wrong spot :) Glad to see the fix on the other PR!